core/src/impl/Kokkos_Core.cpp

```diff
@@ -1165,6 +1165,5 @@ void _kokkos_pgi_compiler_bug_workaround() {}
 #endif
 }  // namespace Kokkos
 
-Kokkos::Impl::InitializationSettingsHelper<std::string>::storage_type const
-    Kokkos::Impl::InitializationSettingsHelper<std::string>::unspecified =
-        "some string we don't expect user would ever provide";
+constexpr char
+    Kokkos::Impl::InitializationSettingsHelper<std::string>::unspecified[];
```

Comment:
Does this need to be declared here at all now?

Reply:
I think so.  This is C++14.